### PR TITLE
Implementing of #154

### PR DESCRIPTION
**Pull request by IremToroslu**
#154 :)

### CODE REVIEW

**Comment by Waldleufer** [comment]
I find the barchart is still pretty slim:

image

I will change that :)

**Comment by IremToroslu** [comment]
image
Mani and I changed the margins and resized the card components in #155 just an fyi as well 😄 And thanks Martin. We tried our best but after somepoint we could only make it this way.

**Comment by Waldleufer** [comment]
now they are :D 

image

**Comment by Waldleufer** [comment]
image

**Review by SaiVarunVaranasi — APPROVED** [review]
Looks good:-)

File: frontend/src/interface/BackendConnect.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ export async function getSimaProProjects() {
  */
 export async function postCalculationRequest(projectId, callback) {
     // Bypass using the SimaPro Data
-    const useDummyData = false;
+    const useDummyData = true;
```
Comment (MartinDuersch) [inline frontend/src/interface/BackendConnect.js:36]: I guess it is better this way haha

Comment (MartinDuersch) [inline frontend/src/interface/BackendConnect.js:36]: I guess it is better this way haha

File: frontend/src/components/details/charts_tables.css [hunk 2]
```diff
@@ -98,16 +104,20 @@
...
     background-color: #ffffff;
-    border-radius: 4px;
+    border-radius: 8px;
     cursor: 'pointer';
     padding: 10px; /* To make it display the shadow properly */
     overflow: auto; /* To make it scrollable whenever it does not fit for some reason. */
+    margin-bottom: 50px;
 }
 
 .TableHeader {
```
Comment (MartinDuersch) [inline frontend/src/components/details/charts_tables.css:123]: Yes margins being margined

**Review by MartinDuersch — APPROVED** [review]

**Pull request merged**